docs/database-engine/availability-groups/windows/basic-availability-groups-always-on-availability-groups.md

```diff
@@ -14,7 +14,7 @@ ms.author: chadam
 # Basic Always On availability groups for a single database
 [!INCLUDE [SQL Server](../../../includes/applies-to-version/sqlserver.md)]
 
-  Always On Basic Availability Groups provide a high availability solution for SQL Server 2016 and SQL Server 2017 Standard Edition. A basic availability group supports a failover environment for a single database. It is created and managed much like traditional (advanced) [Always On Availability Groups &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/always-on-availability-groups-sql-server.md) with Enterprise Edition. The differences and limitations of basic availability groups are summarized in this document.  
+  Always On Basic Availability Groups provide a high availability solution for SQL Server from version 2016 and and above on Standard Edition. A basic availability group supports a failover environment for a single database. It is created and managed much like traditional (advanced) [Always On Availability Groups &#40;SQL Server&#41;](../../../database-engine/availability-groups/windows/always-on-availability-groups-sql-server.md) with Enterprise Edition. The differences and limitations of basic availability groups are summarized in this document.  
```

Comment:
@pituach please update the syntax in this sentence:  Always On Basic Availability Groups provide a high availability solution for SQL Server from version 2016 **and and** above on Standard Edition.

Reply:
I saw this comment only now. The email probably got to the junk folder